src/UI/Actions.hs

```diff
@@ -134,6 +135,33 @@ instance Scrollable 'ScrollingMailView where
 instance Scrollable 'ScrollingHelpView where
   makeViewportScroller _ = Brick.viewportScroll ScrollingHelpView
 
+-- | contexts that have an editor
```

Comment:
Wondering about this attempt, since I think it makes the rendering more explicit when it comes to looking up editors. I started off with a function which looks up the editor by `Name`, but then I need to do the whole pattern matching against other names which are not editors as well. So it's more error prone. Any ideas how this is not a good approach?